src/context/DataContext.jsx

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useState } from 'react';
+
+const DataContext = createContext();
```

Comment:
`createContext` has 1 parameter to set defaultValue of the context

Reply:
Exactly! Why do I need a head?